src/mame/roland/roland_d70.cpp

```diff
@@ -510,8 +513,33 @@ void roland_d70_state::descramble_rom_internal(u8 *dst, const u8 *src) {
 
 ROM_START(d70)
 	ROM_REGION(0x20000, "maincpu", 0)
-	ROM_SYSTEM_BIOS(0, "v203", "Version 2.03")
-	ROMX_LOAD("roland_d70_v110_combined.bin", 0x00000, 0x20000, CRC(52deab1e) SHA1(87d7196888edec65c9feddd16d4c715f6992abc7), ROM_BIOS(0))
```

Comment:
Why was this called “Version 2.03” to begin with?

Reply:
I think wrong copy paste from my side, sorry